docs/content/en/functions/strings.RuneCount.md

```diff
@@ -0,0 +1,28 @@
+---
+title: strings.RuneCount
+description: Determines the number of runes in a string.
+godocref:
+date: 2018-06-01
+publishdate: 2018-06-01
+lastmod: 2018-06-01
+categories: [functions]
+menu:
+  docs:
+    parent: "functions"
+keywords: [counting, character count, length, rune length, rune count]
+signature: ["strings.CountRunes INPUT"]
```

Comment:
RuneCount

Reply:
Bah, was halfway through making it `CountRunes` before I remembered the conflict. Fixed.